brianlong/www.validators.app
# 186628357-add-cache-control-headers

#### What's this PR do?
- set cache control headers for validators/ pages for 5 minutes

#### How should this be manually tested?
- visit validators index or detail page
- check site headers - there should be cache control set for 5 mins (300 seconds)

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186628357)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
